eigen.spec

```diff
@@ -1,21 +1,14 @@
-### RPM external eigen d812f411c3f9
+### RPM external eigen 011e0db31d1bed8b7f73662be6d57d9f30fa457a
```

Comment:
@smuzaffar how did you choose 011e0db31d1bed8b7f73662be6d57d9f30fa457a ?

Reply:
ah, sorry, I missed in your comment before that this comes from tensorflow

could we try also a newer version from the Eigen's master branch ?

Reply:
we can try latest eigen (TF might fail if there are interface changes though)